src/main/java/ch/njol/skript/conditions/CondIsInLove.java

```diff
@@ -0,0 +1,38 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Animals;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Is In Love")
+@Description("Whether or not the animals are currently in the love state.")
```

Comment:
The wording here is a bit strange, i think this would be better:

```suggestion
@Description("Checks whether or not a living entity is in love.")
```